examples/site.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: all
+  roles:
+    - { role: "azavea.yarn" }
+    - { role: "azavea.nodejs", nodejs_version: "6.10.1" }
```

Comment:
I'm a little worried about specifying `nodejs_version` here because in the past we've had difficultly overriding it using `group_vars`. That may not be the case any longer, but I think it would be valuable to leave this responsibility to the role consumer.

We can adds a set of `vars` in the `site.yml` file so ensure that an up-to-date Node.js version is installed.